File: src/main/java/model/Brand.java
package model;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import org.hibernate.annotations.ColumnDefault;
import org.hibernate.annotations.Nationalized;

import java.time.Instant;

@Entity
@Table(name = "Brands")
public class Brand {
    @Id
    @Column(name = "BrandID", nullable = false)
    private Integer id;

    @Nationalized
    @Column(name = "BrandName", nullable = false, length = 100)
    private String brandName;

    @Nationalized
    @Column(name = "Description", length = 500)
    private String description;

    @Nationalized
    @Column(name = "LogoURL")
    private String logoURL;

    @Nationalized
    @Column(name = "Website")
    private String website;

    @ColumnDefault("1")
    @Column(name = "IsActive")
    private Boolean isActive;

    @ColumnDefault("getdate()")
    @Column(name = "CreatedDate")
    private Instant createdDate;

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public String getBrandName() {
        return brandName;
    }

    public void setBrandName(String brandName) {
        this.brandName = brandName;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public String getLogoURL() {
        return logoURL;
    }

    public void setLogoURL(String logoURL) {
        this.logoURL = logoURL;
    }

    public String getWebsite() {
        return website;
    }

    public void setWebsite(String website) {
        this.website = website;
    }

    public Boolean getIsActive() {
        return isActive;
    }

    public void setIsActive(Boolean isActive) {
        this.isActive = isActive;
    }

    public Instant getCreatedDate() {
        return createdDate;
    }

    public void setCreatedDate(Instant createdDate) {
        this.createdDate = createdDate;
    }

}

File: src/main/java/model/Brand.kt
package model

import jakarta.persistence.Column
import jakarta.persistence.Entity
import jakarta.persistence.Id
import jakarta.persistence.Table
import org.hibernate.annotations.ColumnDefault
import org.hibernate.annotations.Nationalized
import java.time.Instant

@Entity
@Table(name = "Brands")
open class Brand {
    @Id
    @Column(name = "BrandID", nullable = false)
    open var id: Int? = null

    @Nationalized
    @Column(name = "BrandName", nullable = false, length = 100)
    open var brandName: String? = null

    @Nationalized
    @Column(name = "Description", length = 500)
    open var description: String? = null

    @Nationalized
    @Column(name = "LogoURL")
    open var logoURL: String? = null

    @Nationalized
    @Column(name = "Website")
    open var website: String? = null

    @ColumnDefault("1")
    @Column(name = "IsActive")
    open var isActive: Boolean? = null

    @ColumnDefault("getdate()")
    @Column(name = "CreatedDate")
    open var createdDate: Instant? = null
}